src/phpDocumentor/Transformer/Template.php

```diff
@@ -124,6 +129,11 @@ public function setVersion(string $version) : void
         $this->version = $version;
     }
 
+    public function files() : FilesystemInterface
```

Comment:
That would add too much business logic to this class as it would act as a wrapper around Flysystem; also: by returning it as a filesystem we can mount it in a mount manager (as seen in the Template Factory, etc). Perhaps the name could be better :)